maps/RadioTower1F.asm

```diff
@@ -1,4 +1,4 @@
-	const_def 2 ; object constants
+RadioTower1FRadioCardWomanOfferQuizText	const_def 2 ; object constants
```

Comment:
This doesn't look right.

Reply:
There's tons of copy-pasting, I must have hit Ctrl+V when I didn't mean to.

Reply:
Thanks.

Reply:
(Actually I think I deleted a newline)

Reply:
Nope, this definitely doesn't belong at the top of the file... Right the first time.